cnivpp/api/bridge/bridge.go

```diff
@@ -49,14 +48,14 @@ func CreateBridge(ch api.Channel, bridgeDomain uint32) error {
 	// Populate the Request Structure
 	req := &l2.BridgeDomainAddDel{
 		BdID:    bridgeDomain,
-		Flood:   1,
-		UuFlood: 1,
-		Forward: 1,
-		Learn:   1,
-		ArpTerm: 0,
+		Flood:   true,
+		UuFlood: true,
+		Forward: true,
+		Learn:   true,
+		ArpTerm: false,
 		MacAge:  0,
```

Comment:
Why isnt MacAge false

Reply:
it's not a boolean, and is implemented as an int.

Reply:
@Eoghan1232  - So this is a mac aging time ? - as opposed to mac aging on/off ?

Reply:
aging time in secs